encuentro-21/ejercitacion_na/paresimpares.js

```diff
@@ -0,0 +1,37 @@
+let numero;
+let cantPar=0;
+let cantImpar=0;
+let total=0;
+let max;
+
+
+function paridad(numero){
+        if(numero % 2 == 0){
+            return true
+        } 
+    return false    
+}
+
+numero =  prompt('Ingrese un numero : ')
+
+
+if (!(Number.isNaN(numero))){
```

Comment:
La function isNaN tambien esta disponible en el contexto global, se podria escribir tambien asi:
```suggestion
if (!(isNaN(numero))){
```

Pero no esta mal poner `Number.isNaN` para que quede explicito que lo sacas del modulo `Number`